lib/components/presentation/templates/nike_zoomer/nike_zoomer_data.dart

```diff
@@ -15,13 +15,16 @@ class NikeZoomerTemplateData {
     code: code,
     id: "nike-zoomer",
     widget: widget,
-    description: 'An animated UI for nike shoes, Animated with a full screen view on mobile and zoom effect on web!',
+    description:
```

Comment:
Maybe using extensions on Strings in a future will help when adding languages.

Reply:
There is an issue to add translations. I think we should focus on solving the perfomance issue. This is really essential for this project and user experience.

Already I appreciate your interest to contributing. If you are interested in the translation, you can look at this [issue](https://github.com/yunweneric/flutter-widgethub/issues/5)